ocs_ci/ocs/acm/acm.py

```diff
@@ -625,3 +629,29 @@ def import_clusters_with_acm():
         )
     else:
         import_clusters_via_cli(clusters)
+
+
+def import_recovery_clusters_with_acm():
+    """
+    Run Procedure of: detecting acm, login to ACM console, import recoevry cluster
+
+    """
+    clusters_env = get_clusters_env()
+    recovery_index = get_recovery_cluster_config().MULTICLUSTER["multicluster_index"]
+    log.info(clusters_env)
+    kubeconfig_recovery = copy_kubeconfig(
+        file=clusters_env.get(f"kubeconfig_location_c{recovery_index}"), return_str=True
+    )
+
+    cluster_name_recoevry = clusters_env.get(f"cluster_name_{recovery_index}")
+    clusters = (cluster_name_recoevry, kubeconfig_recovery)
+    verify_running_acm()
+    if config.DEPLOYMENT.get("ui_acm_import"):
```

Comment:
can remove import via ui, it is not tested in general and cli is fine for now for importing recovery cluster.

Reply:
Removed